_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP? Why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
```

Comment:
`before it's implementation`?
Because we need to omit the usage of "major features" twice.

Reply:
It's getting confusing, reading with `before it's implementation` make me thinking what does `it's` refer to in the sentence